pod/perlapio.pod

```diff
@@ -499,14 +499,14 @@ happened to C<read()> (or whatever) last time IO was requested.
 
 =over 4
 
-=item PerlIO_apply_layers(f,mode,layers)
+=item PerlIO_apply_layers(pTHX_ f,mode,layers)
```

Comment:
This should probably be `aTHX_` since this shows how you're calling the function, not how the function is declared, as in the SYNOPSIS.

Similarly for the `=item` for  PerlIO_binmode().